superset-frontend/src/dashboard/components/SliceHeaderControls/SliceHeaderControls.test.tsx

```diff
@@ -350,3 +353,127 @@ test('Should not show the "Edit chart" without proper permissions', () => {
   });
   expect(screen.queryByText('Edit chart')).not.toBeInTheDocument();
 });
+
+describe('handleDropdownNavigation', () => {
+  const mockToggleDropdown = jest.fn();
+  const mockSetSelectedKeys = jest.fn();
+
+  const menu = React.createElement(
+    'div',
+    { selectedKeys: ['item1'] },
+    React.createElement('div', { key: 'item1' }),
+    React.createElement('div', { key: 'item2' }),
+    React.createElement('div', { key: 'item3' }),
+  );
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('should continue with system tab navigation if dropdown is closed and tab key is pressed', () => {
+    const event = { key: 'Tab', preventDefault: jest.fn() };
+    handleDropdownNavigation(
+      // @ts-ignore
```

Comment:
```suggestion
      // @ts-ignore
```
Not blocking, but... how bad is it? 😰

Reply:
Oh, it didn't like the type of one of the arguments. I think it was the event that I mocked. I could try to redo it if you think it would be better.

Reply:
Well thanks for fixing... I'm not worried about ts-ignore in tests, so much as the actual application. It's another pile I want to track as tech debt when that dashboard is out in public hands :)